arviz/plots/kdeplot.py

```diff
@@ -33,66 +36,85 @@ def kdeplot(values, cumulative=False, rug=False, label=None, bw=4.5, rotated=Fal
     textsize : float
         Size of text on figure.
     plot_kwargs : dict
-        Keywords passed to the pdf line
+        Keywords passed to the pdf line. Ignored for 2D KDE
     fill_kwargs : dict
-        Keywords passed to the fill under the line (use fill_kwargs={'alpha': 0} to disable fill)
+        Keywords passed to the fill under the line (use fill_kwargs={'alpha': 0} to disable fill).
+        Ignored for 2D KDE
     rug_kwargs : dict
-        Keywords passed to the rug plot. Ignored if rug=False
+        Keywords passed to the rug plot. Ignored if rug=False or for 2D KDE
+    countour_kwargs : dict
+        Keywords passed to the countour plot. Ignored for 1D KDE
```

Comment:
Is there a possibility to "remove" the last fill step, so only "contour" lines are filled?